src/test/java/PagesIvaF/LoginPageIvan.java

```diff
@@ -0,0 +1,49 @@
+package PagesIvaF;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+
+public class LoginPageIvan extends BasePageIvan {
+    //Locators
+    @FindBy(css = "[type='email']")
+    WebElement emailField;
+    @FindBy(css = "[type='password']")
+    WebElement  passwordField;
+    @FindBy(css ="[type='submit']" )
```

Comment:
Make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation.

_For, example_

```
    @FindBy(css = "[type='email']")
    private WebElement emailField;
    @FindBy(css = "[type='password']")
    private WebElement  passwordField;
    @FindBy(css ="[type='submit']"
```